duckduckgo/macos-browser
# Update BSK

Task/Issue URL: https://app.asana.com/0/892838074342800/1202795530517724/f
Tech Design URL:
CC:

**Description**:
This PR updates BSK to the latest version with fingerprinting fixes



**Steps to test this PR**:
1. Visit https://privacy-test-pages.glitch.me/privacy-protections/fingerprinting/ and run the tests
1. Expand the results
2. screen.colorDepth and screen.pixelDepth should be **24**
3. screen.availLeft and screen.availTop should be **0**

Additional changes to test:
1. Test the email feature

**Testing checklist**:


---
###### Internal references:
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)
**When ready for review, remember to post the PR in MM**
